Tasks/VsTest/vstest.ts

```diff
@@ -490,10 +496,9 @@ function cleanFiles(responseFile: string, listFile: string, testCaseFilterFile:
     tl.rmRF(testCaseFilterOutput);
 }
 
-function deleteVstestDiagFile(): void {
+function uploadVstestDiagFile(): void {
     if (vstestConfig && vstestConfig.vstestDiagFile && pathExistsAsFile(vstestConfig.vstestDiagFile)) {
-        tl.debug('Deleting vstest diag file ' + vstestConfig.vstestDiagFile);
-        tl.rmRF(vstestConfig.vstestDiagFile);
+        console.log('##vso[task.uploadfile]'+ vstestConfig.vstestDiagFile);
```

Comment:
is there any size restriction for the vstestDiagFile?

Reply:
Can we put a CI event for whether the diag file is created and its size?

Reply:
log message should fine. Ranjana ?